libs/api/domains/education/src/lib/education.service.ts

```diff
@@ -80,19 +79,37 @@ export class EducationService {
       return []
     }
 
-    return family.filter(
-      (familyMember) =>
-        familyMember === myself ||
-        (this.isChild(familyMember) && !this.isChild(myself)),
-    )
+    // Note: we are explicitly sorting by name & national id, since we will
+    // use the index within the family to link to and select the correct
+    // family memember, so that indexes are consistant no matter how they
+    // are displayed or fetched.
+    // They also don't need to be absolute indexes, only indexes that are
+    // unique from the point of view of each viewer.
+
+    return family
+      .filter(
+        (familyMember) =>
+          familyMember === myself ||
+          (this.isChild(familyMember) && !this.isChild(myself)),
```

Comment:
This is kinda hard to read, filter out all other adults in the family and don't show it if I'm <18?

Reply:
yes, you are right.

if the family member we are viewing is a child, and we are an adult, we show allow the family member through.

I updated the code to make it more clear to read, and moved the logic to its own function, and added extra tests